### PR TITLE
Correct formatting, analog to other plurals in article

### PR DESCRIPTION
**Pull request by mexx**

### CODE REVIEW

File: aspnet/fundamentals/logging.rst [hunk 1]
```diff
@@ -143,7 +143,7 @@ First, be sure to add the ``Microsoft.Framework.Logging.TraceSource`` package to
 	:lines: 5-12
 	:emphasize-lines: 8
 
-The following example demonstrates how to configure two separate ``TraceSourceLogger``s for an application, both logging only ``Critical`` messages. Each call to ``AddTraceSource`` takes a ``TraceListener``. The first call configures a ``ConsoleTraceListener``; the second one configures an ``EventLogTraceListener`` to write to the ``Application`` event log. These two listeners are not available in DNX Core, so their configuration is wrapped in a conditional compilation statement (``#if DNX451``).
+The following example demonstrates how to configure two separate ``TraceSourceLoggers`` for an application, both logging only ``Critical`` messages. Each call to ``AddTraceSource`` takes a ``TraceListener``. The first call configures a ``ConsoleTraceListener``; the second one configures an ``EventLogTraceListener`` to write to the ``Application`` event log. These two listeners are not available in DNX Core, so their configuration is wrapped in a conditional compilation statement (``#if DNX451``).
```
Comment (TomArcher) [inline aspnet/fundamentals/logging.rst]: ``TraceSourceLogger`` instances

Comment (mexx) [inline aspnet/fundamentals/logging.rst]: Another variant could be \`\`TraceSourceLogger\`\`\s, which will be rendered as ``TraceSourceLogger``s.

I aimed for consistency :)
Look at the end of line 153:
> two ``TraceListeners``

or line 83:
> ``LogLevels``

I don't know whether there is a guideline about formatting the plurals, but as already say I would go for consistent handling of them in this article.

Which ever way is the preferred, I can update the PR.

Comment (TomArcher) [inline aspnet/fundamentals/logging.rst]: I completely agree with you regarding consistency. My vote would be not to pluralize a class name, but instead to use another word (e.g., instances) to indicate plurality.

**Comment by mexx** [comment]
@TomArcher I've updated the PR.

**Comment by danroth27** [comment]
Thanks! e32a0c12785f9901af9e4072dfb4572d61ed5d13

**Pull request closed without merging**